scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -254,39 +218,47 @@ task LoadData {
 
     echo "Creating the external sample name list table ~{temp_table}"
     bq --apilog=false --project_id=~{project_id} mk ~{temp_table} "sample_name:STRING"
-    NAMES_FILE=~{write_lines(sample_names)}
+
+    NAMES_FILE="sample_names.txt"
+    cut -f 1 ~{bulk_import_fofn} > $NAMES_FILE
+
     bq --apilog=false load --project_id=~{project_id} ~{temp_table} $NAMES_FILE "sample_name:STRING"
 
     # Get the current min/max id, or 0 if there are none. Withdrawn samples still have IDs so don't filter them out.
     bq --apilog=false --project_id=~{project_id} query --format=csv --use_legacy_sql=false ~{bq_labels} '
       SELECT IFNULL(MIN(sample_id),0) as min, IFNULL(MAX(sample_id),0) as max FROM `~{dataset_name}.~{table_name}`
         AS samples JOIN `~{temp_table}` AS temp ON samples.sample_name = temp.sample_name' > results.csv
 
+    SAMPLE_MAP=sample_map.csv
+
     # get sample map of samples that haven't been loaded yet
-    bq --apilog=false --project_id=~{project_id} query --format=csv --use_legacy_sql=false ~{bq_labels} -n ~{num_samples} '
+    num_samples=$(cat ~{bulk_import_fofn} | wc -l)
+    bq --apilog=false --project_id=~{project_id} query --format=csv --use_legacy_sql=false ~{bq_labels} -n $num_samples '
       SELECT sample_id, samples.sample_name FROM `~{dataset_name}.~{table_name}` AS samples JOIN `~{temp_table}` AS temp ON
             samples.sample_name = temp.sample_name WHERE
             samples.sample_id NOT IN (SELECT sample_id FROM `~{dataset_name}.sample_load_status` WHERE status="FINISHED") AND
-            samples.withdrawn is NULL' > sample_map.csv
+            samples.withdrawn is NULL' > $SAMPLE_MAP
 
     ## delete the table that was only needed for this ingest test
     bq --apilog=false --project_id=~{project_id} rm -f=true ~{temp_table}
 
     ## now we want to create a sub list of these samples (without the ones that have already been loaded)
-    curl https://raw.githubusercontent.com/broadinstitute/gatk/ah_var_store/scripts/variantstore/wdl/extract/curate_input_array_files.py -o curate_input_array_files.py
+    curl --location --remote-name https://raw.githubusercontent.com/broadinstitute/gatk/~{branch_name}/scripts/variantstore/wdl/extract/curate_bulk_import_tsv.py
```

Comment:
Wouldn't it be better to just put this python script into the docker itself, rather than using git to pull it from the repo? I think we should limit passing branch names as much as possible in production code.

Reply:
That would require making an additional Variants image based on the GATK image. Not out of the question, but I don't think that's something for this PR.

Reply:
Options to look at:

- Javaizing this and putting it into the GATK image
- Figure out how to get Python script onto GATK image
- Passing it in from a precursor task